models/meirelles2024credit-operations/metadata.ttl

```diff
@@ -0,0 +1,36 @@
+@prefix ontouml: <https://w3id.org/ontouml#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix owl: <http://www.w3.org/2002/07/owl#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix mod: <https://w3id.org/mod#>.
+@prefix dct: <http://purl.org/dc/terms/>.
+@prefix skos: <http://www.w3.org/2004/02/skos/core#>.
+@prefix dcat: <http://www.w3.org/ns/dcat#>.
+@prefix lcc: <http://id.loc.gov/authorities/classification/>.
+
+<https://w3id.org/ontouml-models/model/meirelles2024credit-operations/> a dcat:Dataset, mod:SemanticArtefact;
+    dct:title "Ontologia de Referência para o Domínio de Operações de Crédito";
+    dct:issued "2024"^^xsd:gYear;
+    dcat:theme lcc:H;
+    dct:language "pt-br";
+    dct:license <https://creativecommons.org/licenses/by/4.0/>;
+    dct:contributor <https://orcid.org/0009-0003-8838-3367>;
+    mod:designedForTask ontouml:ConceptualClarification, ontouml:Interoperability, ontouml:OntologicalAnalysis;
+    ontouml:context ontouml:Research;
+    dct:source <http://repositorio.ufes.br/handle/10/18082>.
+<https://w3id.org/ontouml-models/model/meirelles2024credit-operations/> dcat:distribution <https://w3id.org/ontouml-models/model/meirelles2024credit-operations/distribution/ttl>.
+<https://w3id.org/ontouml-models/model/meirelles2024credit-operations/distribution/ttl> a dcat:Distribution;
+    dct:title "Turtle distribution of \"Ontologia de Referência para o Domínio de Operações de Crédito\""@en;
+    dcat:mediaType <https://www.iana.org/assignments/media-types/text/turtle>;
+    dcat:downloadURL <https://raw.githubusercontent.com/unibz-core/ontouml-models/master/meirelles2024credit-operations/ontology.ttl>.
+<https://w3id.org/ontouml-models/model/meirelles2024credit-operations/> dcat:distribution <https://w3id.org/ontouml-models/model/meirelles2024credit-operations/distribution/ttl>.
+<https://w3id.org/ontouml-models/model/meirelles2024credit-operations/distribution/ttl> a dcat:Distribution;
+    dct:title "JSON distribution of \"Ontologia de Referência para o Domínio de Operações de Crédito\""@en;
+    dcat:mediaType <https://www.iana.org/assignments/media-types/application/json>;
+    dcat:downloadURL <https://raw.githubusercontent.com/unibz-core/ontouml-models/master/meirelles2024credit-operations/ontology.json>.
+<https://w3id.org/ontouml-models/model/meirelles2024credit-operations/> dcat:distribution <https://w3id.org/ontouml-models/model/meirelles2024credit-operations/distribution/ttl>.
+<https://w3id.org/ontouml-models/model/meirelles2024credit-operations/distribution/ttl> a dcat:Distribution;
```

Comment:
There is a problem here. Shouldn't this be the Visual Paradigm distribution of the ontology? The URI and the title are wrong.